Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -301,10 +301,12 @@ public function add($dn, array $entry)
         return $this;
     }
 
-    /** Alias of self::unbind() */
+    /**
+     * Alias of self::unbind()
+     */
     public function close()
     {
-        return $this->unbind();
+        $this->unbind();
```

Comment:
Personally I would then update `close()` method accordingly. You will still need to do that if you decide to add some arguments to the method. Please confirm.